### ISSUE TITLE
Feauture: Display Abastract syntax tree

### ISSUE THREAD

**Issue by Stephanevg** [Stephanevg/PSClassUtils#67]
Consider creating an abastract syntax tree visulisation for scripts:

some information on AST:
https://geekeefy.wordpress.com/2017/06/07/powershell-tokenization-and-abstract-syntax-tree/

**Comment by PeterMTaylor** [comment]
Good idea!  If the recursive option was used that is reading a directory of scripts, be better to display one image per file script otherwise gets clustered and too hard to see.

**Comment by Stephanevg** [comment]
@PeterMTaylor 
What do you think of:

```
-OutputType [Combined/Single]
```

?

**Comment by PeterMTaylor** [comment]
Sounds good. Go for it! Merry Xmas too!
